security/cmd/node_agent/main.go

```diff
@@ -69,6 +71,8 @@ func init() {
 		"key", "/etc/certs/key.pem", "Node Agent private key file")
 	flags.StringVar(&cAClientConfig.RootCertFile, "root-cert",
 		"/etc/certs/root-cert.pem", "Root Certificate file")
+	flags.StringVar(&sanTypeArg, "san-type",
```

Comment:
Can we make this more general, by including using CN as identity to as well?

Reply:
Done.